app/views/declaration/destinationCountries/destination_country.scala.html

```diff
@@ -15,39 +15,43 @@
  *@
 
 @import controllers.declaration.routes
-@import forms.declaration.destinationCountries.DestinationCountries
 @import models.requests.JourneyRequest
 @import services.Countries
 @import services.view.AutoCompleteItem
-@import uk.gov.hmrc.play.views.html._
 @import views.Title
 
 @this(main_template: views.html.main_template)
 
-@(mode: Mode, form: Form[DestinationCountries])(implicit request: JourneyRequest[_], messages: Messages)
+@(mode: Mode, form: Form[String])(implicit request: JourneyRequest[AnyContent], messages: Messages)
 
 @main_template(
-    title = Title("declaration.destinationCountries.title", "declaration.summary.locations.header")
+    title = Title("declaration.destinationCountry.title", "declaration.destinationCountry.heading")
 ) {
-    @components.back_link(controllers.declaration.routes.OriginationCountryController.displayPage(mode), mode)
+
+    @if(request.isType(DeclarationType.SIMPLIFIED)) {
+        @components.back_link(routes.DeclarationHolderController.displayPage(), mode)
```

Comment:
Please move this logic to Navigator companion object

Reply:
Done